doc/api/crypto.md

```diff
@@ -2691,9 +2691,11 @@ changes:
     description: The `buffer` argument may be any `TypedArray` or `DataView`.
 -->
 
-* `buffer` {Buffer|TypedArray|DataView} Must be supplied.
+* `buffer` {Buffer|TypedArray|DataView} Must be supplied. The
+  size of the provided `buffer` must not be larger than `2**31 - 1`.
```

Comment:
Ditto
```suggestion
  size of the provided `buffer` must be less or equal to `2**31 - 1`.
```

Reply:
FWIW, I think `less than or equal to` would be the way to go here and in the other three examples, but non-blocking on whatever language is chosen.

```suggestion
  size of the provided `buffer` must be less than or equal to `2**31 - 1`.
```